api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -268,6 +301,32 @@ private String getFreeCreditsResolutionText() {
         .build();
   }
 
+  private ImmutableMap<EmailSubstitutionField, String> setupBillingAccountEmailMap(
+      DbUser user, SendBillingSetupEmailRequest request) {
+
+    return new ImmutableMap.Builder<EmailSubstitutionField, String>()
+        .put(EmailSubstitutionField.HEADER_IMG, getAllOfUsLogo())
+        .put(EmailSubstitutionField.FIRST_NAME, user.getGivenName())
+        .put(EmailSubstitutionField.LAST_NAME, user.getFamilyName())
+        .put(EmailSubstitutionField.ALL_OF_US, getAllOfUsItalicsText())
+        .put(EmailSubstitutionField.INSTITUTION_NAME, request.getInstitution())
+        .put(EmailSubstitutionField.USER_PHONE, request.getPhone())
```

Comment:
these are untrusted inputs, please escape - see example / comment above: https://github.com/all-of-us/workbench/blob/ede45de8545a577f85d651a0a3d573ec11046228/api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java#L103